bcachefs/utils.h

```diff
@@ -0,0 +1,38 @@
+/* Include Guard */
+#ifndef INCLUDE_UTILS_H
+#define INCLUDE_UTILS_H
+
+/**
+ * Includes
+ */
+
+#include <stdio.h>
+
+/* Extern "C" Guard */
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef signed char int8_t;
```

Comment:
Peut-être `#include <stdint.h>`? Si tu utilises des features de C99 de toute façon comme les designated initializers, c'est un gage que `stdint.h` est disponible aussi, et qu'on peut aussi tasser de côté MSVC < 2015 si on avait des rêves de supporter Windows. Et puis mon implémentation de SipHash se sert de `stdint.h` aussi.

Reply:
Good point. See this for the fix: https://github.com/mila-iqia/bcachefs/pull/25#discussion_r744971177